test/realm_test.dart

```diff
@@ -1859,6 +1859,46 @@ Future<void> main([List<String>? args]) async {
     expect(realmCore.getDeviceName(), matcher);
     expect(realmCore.getDeviceVersion(), matcher);
   });
+
+  test('Realm path with unicode symbols', () {
+    var config = Configuration.local([Car.schema], path: "${generateRandomUnicodeString(10)}.realm");
```

Comment:
This should just call generateRandomEmail passing an optional argument for including unicode symbols or not